ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -2771,5 +2771,50 @@
     "name": "type-blank.svg",
     "mediaType": "image/svg+xml"
   }
- }
+  },
+  {
+    "id": "openshift",
+    "name": "openshift",
+    "description": "Openshift Workspace",
+    "scope": "general",
+    "tags": [
+      "Openshift"
+    ],
+    "workspaceConfig": {
+      "environments": {
+        "default": {
+          "machines": {
+            "dev-machine/main": {
+              "installers": [
+                "org.eclipse.che.exec", "org.eclipse.che.terminal", "org.eclipse.che.ws-agent"
+              ],
+              "servers": {},
+              "attributes" : {
+                "memoryLimitBytes": "2147483648"
+              }
+            }
+          },
+          "recipe": {
```

Comment:
I'm just saying that che is a 'portable workspace' so I was expecting I could use my workspace on all platforms whatever is their native stuff. It's like when I use docker I don't care if it's windows, linux or MacOS as native platform. 
If I need to rewrite the data of my workspace then it's not anymore 'portable'